streamwebs_frontend/streamwebs/tests/models/test_wq_sample.py

```diff
@@ -91,8 +91,11 @@ def test_optional_fields(self):
             self.assertEqual(
                 sample._meta.get_field(field).blank, True)
 
-    def test_Sample_ManyToOneTransect(self):
-        sample_1 = WQ_Sample.objects.create_sample(self.water_quality)
+    def test_Sample_ManyToOneWaterQuality(self):
+        sample_1 = WQ_Sample.objects.create_sample(self.water_quality, 0, 0,
+                                                   0, 0, 0, 0,
```

Comment:
Nice work-- since you've now changed your managers, you can consider varying your fake data to simulate real data, i.e. instead of passing in all 0s for your required fields, pass in other numbers and assert them.